saleor/core/utils/anonymization.py

```diff
@@ -0,0 +1,71 @@
+import copy
+from typing import TYPE_CHECKING
+
+from faker import Faker
+
+from ...account.models import Address, User
+
+if TYPE_CHECKING:
+    from ...order.models import Order
+    from ...checkout.models import Checkout
+
+fake = Faker()
+
+
+def generate_fake_address() -> "Address":
+    return Address(
+        first_name=fake.first_name(),
+        last_name=fake.last_name(),
+        company_name=fake.company(),
+        street_address_1=fake.street_address(),
+        street_address_2=fake.secondary_address(),
+        city=fake.city(),
+        postal_code=fake.postalcode(),
+        country=fake.country(),
+        phone=fake.phone_number(),
+    )
+
+
+def generate_fake_user() -> "User":
+    return User(
+        email=fake.email(),
+        first_name=fake.first_name(),
+        last_name=fake.last_name(),
+        is_active=True,
+        note=fake.paragraph(),
+        date_joined=fake.date_time(),
+        default_shipping_address=generate_fake_address(),
+        default_billing_address=generate_fake_address(),
+        meta=fake.pystruct(count=1),
+        private_meta=fake.pystruct(count=1),
+    )
+
+
+def anonymize_order(order: "Order"):
```

Comment:
For anonymization itself, we also have a module already - `saleor/extensions/plugins/anonymize/__init__.py` - it's a plugin which we use to anonymize checkouts in the demo store. I'm not sure if we can reuse those functions in those two places, but we could try to have less code duplication.

Reply:
The `anonymize` plugin also differs in that it saves the changes and it _obfuscate_ rather than generates _fake_ data.

I'm open for suggestions. Perhaps some name changing could clarify things. For me these modules perform similar but not identical functionality and I don't see how to merge them.